source/transfer_list.rst

```diff
@@ -819,6 +819,46 @@ and reserved for use as a TPM Command Response Buffer interface.
      - hdr_size + 0x8
      - Size of CRB.
 
+Cryptogragphy library heap information (XFERLIST_CRYPTO_LIB_HEAP_INFO)
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+The cryptograph library heap info specifies the base address and size of a
+region of memory on the heap that has been carved out for use by a cryptography
+library.
+
```

Comment:
I don't think the description is enough. Some information needs to be added on how the subsequent layers are supposed to use that information. e.g do they have to reserve that memory when they define their heap etc.

Reply:
+1

Also, to me, 'heap' suggests some sort of memory allocation going on. Is this in a particular format? How would someone set it up? Is this targeted at a particular library? If so you could reference it here.

Reply:
Thank you both for your valuable feedback!

Our primary focus is on the Mbed-TLS library. At this point, I haven’t found any other libraries where this information might be pertinent. Given this, I’ve refined the scope of the TE to exclusively cover Mbed-TLS and relocated the TE within TF-A’s range. Please share your thoughts if you think there’s potential for the TE in the generic range.

Additionally, I’ve enhanced the explanation regarding the expected usage of memory by the producer and consumer. This should provide a clearer understanding of how this TE should be used.